File: SEM3_CODE/PHP/BASIC/09_largest_two.php
<?php
$firstNumber = (float) readline('Enter first number: ');    // Read the first number
$secondNumber = (float) readline('Enter second number: ');  // Read the second number

if ($firstNumber >= $secondNumber) {                        // Compare both numbers
    $largest = $firstNumber;                                // Store the first number
} else {
    $largest = $secondNumber;                               // Store the second number
}

echo 'Largest number: ' . $largest;                         // Display the largest number
?>

File: SEM5_CODE/PYTHON/BASIC/09_largest_two.py
first_number = float(input("Enter first number: "))    # Read the first number
second_number = float(input("Enter second number: "))  # Read the second number

if first_number >= second_number:                      # Compare both numbers
    largest = first_number                             # Store the first number
else:
    largest = second_number                            # Store the second number

print("Largest number:", largest)                      # Display the largest number
